src/components/Counter.js

```diff
@@ -56,4 +56,4 @@ const mapStateToProps = (state) => {
 // is only a dumb React component. We pass in all of the functions that
 // are reliant on Redux, along with the component itself, so that Redux
 // makes itself known to this component.
-export default connect(mapStateToProps, { increment, decrement })(Counter);
+export default connect(mapStateToProps, { increment:increment, decrement:decrement })(Counter);
```

Comment:
On Line 59, if the key is equal to the value then you can just return the key. For example {increment:increment} can be {increment}